crates/ln-dlc-node/src/disk.rs

```diff
@@ -15,13 +15,13 @@ pub(crate) fn read_scorer(
     logger: Arc<TracingLogger>,
 ) -> ProbabilisticScorer<Arc<NetworkGraph>, Arc<TracingLogger>> {
     let params = ProbabilisticScoringParameters::default();
-    if let Ok(file) = File::open(path) {
-        let args = (params.clone(), graph.clone(), logger.clone());
-        match ProbabilisticScorer::read(&mut BufReader::new(file), args) {
-            Ok(scorer) => return scorer,
-            Err(e) => tracing::error!("Failed to read scorer from disk: {e}"),
-        }
-    }
+    // if let Ok(file) = File::open(path) {
```

Comment:
nit: I guess we can remove that commented code and if we need it again we can look at this commit.

Reply:
I think the second commit is more serious. Unfortunately I can't reproduce the panic anymore :/